src/WhatToCook.WebApp/ClientApp/src/app/recipes/recipe-view/recipe-view.component.html

```diff
@@ -0,0 +1,119 @@
+<div [ngSwitch]="getDisplayMode()">
+  <div *ngSwitchCase="0">
+    <ng-container *ngTemplateOutlet="editable"></ng-container>
+  </div>
+  <div *ngSwitchCase="1">
+    <ng-container *ngTemplateOutlet="editable"></ng-container>
+  </div>
+  <div *ngSwitchCase="2">
+    <ng-container *ngTemplateOutlet="view"></ng-container>
+  </div>
+</div>
+//TODO:  a pleasant css for each category: name, description etc.
```

Comment:
Unresolved todo

Reply:
deleted TODO, will be in the near future